ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorDemoController.swift

```diff
@@ -7,84 +7,157 @@ import FluentUI
+            DemoColorItem(text: Colors.Palette.orange20.name, color: Colors.Palette.orange20.color),
+            DemoColorItem(text: Colors.Palette.orangeYellow20.name, color: Colors.Palette.orangeYellow20.color),
+            DemoColorItem(text: Colors.Palette.green20.name, color: Colors.Palette.green20.color),
+            DemoColorItem(text: Colors.Palette.green10.name, color: Colors.Palette.green10.color),
+            DemoColorItem(text: Colors.Palette.cyan30.name, color: Colors.Palette.cyan30.color),
+            DemoColorItem(text: Colors.Palette.cyan20.name, color: Colors.Palette.cyan20.color),
+            DemoColorItem(text: Colors.Palette.cyanBlue20.name, color: Colors.Palette.cyanBlue20.color),
+            DemoColorItem(text: Colors.Palette.blue10.name, color: Colors.Palette.blue10.color),
+            DemoColorItem(text: Colors.Palette.blueMagenta30.name, color: Colors.Palette.blueMagenta30.color),
+            DemoColorItem(text: Colors.Palette.blueMagenta20.name, color: Colors.Palette.blueMagenta20.color),
+            DemoColorItem(text: Colors.Palette.magenta20.name, color: Colors.Palette.magenta20.color),
+            DemoColorItem(text: Colors.Palette.magenta10.name, color: Colors.Palette.magenta10.color),
+            DemoColorItem(text: Colors.Palette.magentaPink10.name, color: Colors.Palette.magentaPink10.color),
+            DemoColorItem(text: Colors.Palette.gray40.name, color: Colors.Palette.gray40.color),
+            DemoColorItem(text: Colors.Palette.gray30.name, color: Colors.Palette.gray30.color),
+            DemoColorItem(text: Colors.Palette.gray20.name, color: Colors.Palette.gray20.color)
+        ]),
+        DemoColorSection(text: "Message colors", items: [
+            DemoColorItem(text: Colors.Palette.dangerPrimary.name, color: Colors.Palette.dangerPrimary.color),
+            DemoColorItem(text: Colors.Palette.dangerTint40.name, color: Colors.Palette.dangerTint40.color),
+            DemoColorItem(text: Colors.Palette.dangerTint10.name, color: Colors.Palette.dangerTint10.color),
+            DemoColorItem(text: Colors.Palette.dangerShade40.name, color: Colors.Palette.dangerShade40.color),
+            DemoColorItem(text: Colors.Palette.dangerShade10.name, color: Colors.Palette.dangerShade10.color),
+            DemoColorItem(text: Colors.Palette.warningPrimary.name, color: Colors.Palette.warningPrimary.color),
+            DemoColorItem(text: Colors.Palette.warningTint40.name, color: Colors.Palette.warningTint40.color),
+            DemoColorItem(text: Colors.Palette.warningTint10.name, color: Colors.Palette.warningTint10.color),
+            DemoColorItem(text: Colors.Palette.warningShade40.name, color: Colors.Palette.warningShade40.color),
+            DemoColorItem(text: Colors.Palette.warningShade30.name, color: Colors.Palette.warningShade30.color)
+        ])
+    ]
+    private var tableView: UITableView!
+    private var usingDefaultPrimaryColor: Bool = false
+
+    override func viewDidLoad() {
```

Comment:
This looks like it's copied code, but easy to fix: this looks like a loadView rather than viewDidLoad. viewDidLoad is used for additional configuration after loading from a xib. loadView is where programmatic hierarchy is created and usually culminates in a "view = " call. See hopefully https://github.com/anandrajeswaran/apple-ux-guide/blob/master/ViewControllers.md later today :)